packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -161,7 +161,8 @@ export function createTrackWire(property: keyof Tagging): Wire<Taggable> {
       taggingInfo.params.location = location as string;
       return taggingInfo;
     }),
-    ({ eventPayload: { tagging } }) => !!tagging?.[property]
+    ({ eventPayload: { tagging }, metadata: { ignoreInModules } }) =>
+      !!tagging?.[property] && !ignoreInModules?.includes(moduleName)
```

Comment:
Should we do this in the `wires.factory.ts` to make it compatible with all the modules?
Maybe we can explore it in a future refactor.

Reply:
I thought about it but right now is better to do it in a case by case basis for now. We would add an if statement to all the wires and only one will use it. Better keeping it in mind for the future if we have more cases.